src/extension.ts

```diff
@@ -184,8 +185,11 @@ function registerDockerClients(): void {
         ext.orchestratorManager.registerRuntimeClient(composeClient)
     );
 
+    // Sets context value for whether Docker version supports sbom
+    await checkDockerVersionForSbom();
```

Comment:
IMO it's not worth slowing the activation to prevent this command from showing up on unsupported Docker versions. I'd be fine with just trying the command and showing an error if it doesn't work for any reason.